GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -149,7 +149,7 @@ void DrawEngineVulkan::InitDeviceObjects() {
 	VkDescriptorPoolSize dpTypes[2];
 	dpTypes[0].descriptorCount = 2048;
 	dpTypes[0].type = VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER_DYNAMIC;
-	dpTypes[1].descriptorCount = 2048;
+	dpTypes[1].descriptorCount = 4096;
```

Comment:
Can/should we add the current count or current max to debug statistics?

-[Unknown]